discovery_engine/test/discovery_engine/discovery_engine_worker_test.dart

```diff
@@ -43,8 +39,8 @@ void main() {
     });
 
     test(
-        'when sending "FeedRequested" event as payload it should respond with '
-        '"FeedRequestSucceeded" event', () async {
```

Comment:
The previous test didn't make too much sense in the first place, because some of the things in the worker should be mocked, but there is no easy way to do it in this context. It was working cause previously we replied with empty events, without really handling them. So it actualy made sense to remove it.